homeassistant/components/mill/sensor.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, heater, mill_data_connection, sensor_type):
         self._attr_name = f"{heater.name} {sensor_type.replace('_', ' ')}"
         self._attr_unique_id = f"{heater.device_id}_{sensor_type}"
         self._attr_native_unit_of_measurement = ENERGY_KILO_WATT_HOUR
-        self._attr_state_class = STATE_CLASS_MEASUREMENT
+        self._attr_state_class = STATE_CLASS_TOTAL
```

Comment:
This, `_attr_native_unit_of_measurement` and `_attr_device_class` are static and should be defined as class attributes on the `MillHeaterEnergySensor` class

Reply:
And `_attr_device_class` :)